### PR TITLE
[velero] Update owners

### PR DESCRIPTION
**Pull request by jenting**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] ~Variables are documented in the values.yaml or README.md~
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

File: charts/velero/Chart.yaml
Comment (zubron) [inline charts/velero/Chart.yaml:11]: Should this be my github username? In that case, it should be `zubron`.

**Review by zubron — CHANGES_REQUESTED** [review]

**Review by zubron — COMMENTED** [review]
LGTM! I'll review again once it's been rebased. Thanks :)

**Review by zubron — COMMENTED** [review]
The change looks fine but I realised we're now at two maintainers, and you are the most significant contributor, @jenting! This will make it difficult for you to get changes merged. Can we add one of the other Velero maintainers here? @dsu-igeek @sseago

**Comment by zubron** [comment]
We can add @reasonerjt and @ywk253100 into the list as well.

**Review by zubron — APPROVED** [review]
Thanks, @jenting!

**Review by sseago — APPROVED** [review]

**Pull request merged**
